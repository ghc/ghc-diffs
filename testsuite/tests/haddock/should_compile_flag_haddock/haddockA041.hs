-- | File starts with a Haddock comment
main :: IO ()
main = pure ()
