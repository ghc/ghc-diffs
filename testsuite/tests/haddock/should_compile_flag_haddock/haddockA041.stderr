
==================== Parser ====================
<document comment>
main :: IO ()
main = pure ()
